src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -167,4 +167,8 @@
   border: var(--pf-c-clipboard-copy__expandable-content--BorderWidth) solid transparent;
   outline-offset: var(--pf-c-clipboard-copy__expandable-content--OutlineOffset);
   box-shadow: var(--pf-c-clipboard-copy__expandable-content--BoxShadow);
+
+  > pre {
```

Comment:
is the `>` necessary? That causes problems if react or the user introduces another element for whatever reason.